src/perf/lib/ThroughputClient.cpp

```diff
@@ -0,0 +1,410 @@
+/*++
+
+    Copyright (c) Microsoft Corporation.
+    Licensed under the MIT License.
+
+Abstract:
+
+    QUIC Perf Throughput Client Implementation.
+
+--*/
+
+#ifdef QUIC_CLOG
+#include "ThroughputClient.cpp.clog.h"
+#endif
+
+#ifndef _KERNEL_MODE
+#define QUIC_TEST_APIS 1
```

Comment:
Why does client need QUIC_TEST_APIS?

Reply:
It doesn't, but kernel mode does need quic_platform. So did just that by itself instead.